ci/testnet-deploy.sh

```diff
@@ -338,6 +338,11 @@ if ! $skipCreate; then
     create_args+=($maybeValidatorAdditionalDiskSize)
   fi
 
+  # Disable GCE self-destruct
+  if [[ "$cloudProvider" == "gce" ]]; then
+    create_args+=( --self-destruct-minutes 0 )
```

Comment:
nit: if we just support `--self-destruct-minutes 0` for all providers then the gce check here goes away